scripts/get_message_templates.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+path="$1"
+function getMessageTemplates() {
+  echo
+  echo
+  grep -R "newReportNode().withMessageTemplate" "$path" | awk -F '"' '{print $2, $3, $4}' | sort -g | uniq -c | sed "s/,/=/g" > message_templates.txt
```

Comment:
Could you also rename the generated file, for instance in "dictionary.properties", and add it in the `.gitignore" (to avoid committing it)?

For instance:
```
**/dictionary.properties
```